packages/field-plugin/src/createFieldPlugin/createPluginActions/createPluginMessageListener/handlePluginMessage.test.ts

```diff
@@ -123,10 +134,34 @@ describe('handlePluginMessage', () => {
     }
     const callbacks = mockCallbacks()
     handlePluginMessage(data, uid, callbacks)
+    expect(callbacks.onLoaded).not.toHaveBeenCalled()
     expect(callbacks.onStateChange).not.toHaveBeenCalled()
     expect(callbacks.onContextRequest).not.toHaveBeenCalled()
     expect(callbacks.onUserContextRequest).not.toHaveBeenCalled()
     expect(callbacks.onAssetSelect).toHaveBeenCalledWith(data)
+    expect(callbacks.onPromptAI).not.toHaveBeenCalled()
+    expect(callbacks.onUnknownMessage).not.toHaveBeenCalled()
+  })
+
+  it('handles promptAI messages', () => {
+    const data: PromptAIResponseMessage = {
+      action: 'prompt-ai',
+      uid,
+      callbackId: 'test-callback-id',
+      aiResponse: {
+        ok: true,
+        answer: 'Some AI generated text',
+      },
+    }
+
+    const callbacks = mockCallbacks()
+
+    handlePluginMessage(data, uid, callbacks)
+    expect(callbacks.onLoaded).not.toHaveBeenCalled()
+    expect(callbacks.onStateChange).not.toHaveBeenCalled()
+    expect(callbacks.onContextRequest).not.toHaveBeenCalled()
+    expect(callbacks.onAssetSelect).not.toHaveBeenCalledWith(data)
+    expect(callbacks.onPromptAI).toHaveBeenCalled()
```

Comment:
I guess you should interchange these two?

```suggestion
    expect(callbacks.onAssetSelect).not.toHaveBeenCalled()
    expect(callbacks.onPromptAI).toHaveBeenCalledWith(data)
```

Reply:
Great catch!! Done https://github.com/storyblok/field-plugin/pull/448/commits/3f85b249aae7cb6bfac42a7df2b300b8fe810c46 ❤️